enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1674,5 +1818,13 @@ func (p *PebbleCache) Stop() error {
 		return err
 	}
 
+	p.closedMu.Lock()
+	defer p.closedMu.Unlock()
+	if p.closed {
+		return nil
+	}
+	p.closed = true
+	p.dbWaiters.Wait() // wait for all db users to finish up.
```

Comment:
Can you add a log before & after the wait so we can tell when the shutdown began & ended?

Reply:
PTAL